.virtualenv.requirements.txt

```diff
@@ -11,6 +11,9 @@ lxml
 # YAML file parsing
 PyYAML
 
+# TOML file parsing
+toml
```

Comment:
This module is dead. It was split into [`tomllib` in the standard library](https://docs.python.org/3/library/tomllib.html) for reading, and `tomli-w` for writing.

If we want a singular module for this, `tomlkit` is a better choice, and it's style preserving too.

Reply:
It's there because anymarkup uses it as its backend.

Reply:
kiwi depends on anymarkup because it can be feed with any markup and can translate both sides. So if you want tomlkit rather than toml it should be added to anymarkup not kiwi

Reply:
That's a good idea.

Reply:
I opened an issue upstream, let's see how they feel about it